### PR TITLE
Till quickstart bear

### PR DESCRIPTION
**Pull request by ishanSrt**

### CODE REVIEW

**Comment by satwikkansal** [comment]
ack 9ae74d4 ea62d33 d9466b7 ecdb90c

**Comment by satwikkansal** [comment]
@gitmate-bot ff

File: coala_quickstart/generation/Utilities.py [hunk 1]
```diff
@@ -187,3 +188,51 @@ def get_language_from_hashbang(hashbang):
...
             hashbang_element = hashbang_contents[0].split('/')
             return (hashbang_element[len(hashbang_element)-1])
+
+
+def concatenate(dict1, dict2):
+    """
+    Concatenates 2 dicts of the type:
+    eg.
+
+    dict1:
```
Comment (jayvdb) [inline coala_quickstart/generation/Utilities.py]: this needs to be a doctest.

c.f. https://github.com/coala/coala-quickstart/pull/263#discussion_r208909218

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-quickstart/pull/286/commits/468893fe4c6b64882a6e1b63b8edfac6ac18dd2c

**Comment by jayvdb** [comment]
Looks like Windows is timing out.  Does it need a longer timeout ?
Or something else is wrong ?

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-quickstart/pull/286/commits/9354c3062cef7f4309c801870e172d3ff180440c

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**